KillTeam/Commands/Handlers/IHandleCommands.cs

```diff
@@ -0,0 +1,7 @@
+namespace KillTeam.Commands.Handlers
+{
+    public interface IHandleCommands<TCommand>
+    {
+        void Handle(TCommand command);
+    }
+}
```

Comment:
I like this concept. Like reusable actions.

Reply:
Yeah at work all our API microservices use CQRS. This is the beginning of moving towards that. Next step is to start moving queries out of Pages and ViewModels.

End goal is to KTContext be able to spin up an in-memory at least local DB for tests so this has even more benefit.